.github/workflows/connectors.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       - name: Install Rust
         uses: actions-rs/toolchain@v1
         with:
-          toolchain: nightly
+          toolchain: nightly-2023-04-16
```

Comment:
is nightly still required?

Reply:
@jgraettinger yes, the ICE is fixed in 1.70, but it's not released as stable yet.